### PR TITLE
added the touch events + removed the weired effect while focus#341

### PR DESCRIPTION
**Pull request by mahmoud-zakria-sociomantic**

### CODE REVIEW

File: src/Slider/slider.css [hunk 1]
```diff
@@ -60,6 +60,7 @@
 
         user-select:    none;
         cursor:         pointer;
+        -webkit-tap-highlight-color: transparent;
```
Comment (conor-cafferkey-sociomantic) [inline src/Slider/slider.css]: Please make sure that indentation is kept consistent

Comment (conor-cafferkey-sociomantic) [inline src/Slider/slider.css]: Please make sure that indentation is kept consistent

File: src/Slider/index.jsx [hunk 2]
```diff
@@ -447,29 +447,31 @@ export default class Slider extends Component
             return;
         }
 
-        event.preventDefault();
-
         const { index } = event.target.dataset;
 
+        event.stopPropagation();
```
Comment (conor-cafferkey-sociomantic) [inline src/Slider/index.jsx]: I think we probably don't need this here

Comment (mahmoud-zakria-sociomantic) [reply src/Slider/index.jsx]: Done

File: src/Slider/slider.css (hunk 1 above)
Comment (mahmoud-zakria-sociomantic) [reply src/Slider/slider.css]: Done

**Pull request merged**